<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="data_fields_layout">
     <item>
      <layout class="QHBoxLayout" name="sampleLayout">
       <item>
        <widget class="QLineEdit" name="sample_path"/>
       </item>
       <item>
        <widget class="QPushButton" name="sample_button">
         <property name="text">
          <string>Sample</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="dark_flat_button">
       <property name="text">
        <string>Dark &amp;&amp; Flat?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="flat_widget" native="true">
       <layout class="QHBoxLayout" name="flatLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="flat_path"/>
        </item>
        <item>
         <widget class="QPushButton" name="flat_button">
          <property name="text">
           <string>Flat</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="dark_widget" native="true">
       <layout class="QHBoxLayout" name="darkLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="dark_path"/>
        </item>
        <item>
         <widget class="QPushButton" name="dark_button">
          <property name="text">
           <string>Dark</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="index_start">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>End</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="index_end">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Step</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="index_step">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="step_preview">
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="step_all">
         <property name="text">
          <string>All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="expectedResourcesLabel">
     <property name="text">
      <string>Placeholder txt for Developer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Custom Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="stackName">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set custom name that will be used as the stack's name. If not provided the folder name will be used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="parallelLoad">
       <property name="text">
        <string>Parallel Load</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Pixel Bit Depth:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="pixelBitDepth">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
